### PR TITLE
Move to 3.0 Preview 6 SDK

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -38,4 +38,11 @@
     -->
     <MicrosoftNETCoreAppRuntimeVersion>$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)</MicrosoftNETCoreAppRuntimeVersion>
   </PropertyGroup>
+  <!-- Restore feeds -->
+  <PropertyGroup Label="Restore feeds">
+    <RestoreSources Condition=" '$(DotNetBuildOffline)' != 'true' ">
```
Comment (bricelam) [inline eng/Versions.props]: Is this for source build? We don't have to do that for EF6, do we?

Comment (dougbu) [reply eng/Versions.props]: Source build isn't a direct issue for EF 6 or EF Core but it is an issue to either (a) leave our builds behind on older SDKs or (b) get to far ahead of where AspNetCore is in EF Core.

Comment (dougbu) [reply eng/Versions.props]: And, these extra restore sources are necessary only because the Preview 6 SDK no longer implicitly adds our internal feeds.

Comment (bricelam) [reply eng/Versions.props]: But the condition on this line is just dead code, right?

Comment (dougbu) [reply eng/Versions.props]: No, it's the pattern we need to put in place for servicing builds et cetera.

**Pull request merged**